### ISSUE TITLE
Allow "worldCamVisibility" property for Attached cameras

### ISSUE THREAD

**Issue by blackorbit1** [kinsi55/CS_BeatSaber_Camera2#88]
From what I've seen on twitch and what I've experienced myself, most of the time the Attached camera is used as a 3rd person camera with a smooth following. 
The issue is that ingame you really have a vague idea of where the camera is in this mode and you don't really know where to look at if you want to interact with the audience, or even if you're wondering what the camera sees for testing purpose.

Here's an idea of what it could look like ingame:
![image](https://user-images.githubusercontent.com/38140797/190130546-b2bbce7a-e963-4e98-8dcb-24b9bbfc7a9a.png)

**Comment by kinsi55** [comment]
With the Upcoming update, Attached cameras wont really be used any more for this usecase (Mawntee for example now uses [the new System](https://github.com/kinsi55/CS_BeatSaber_Camera2/commit/5ba4142d9d99c70a70e1b8f5b4d47938984db774))

**Comment by blackorbit1** [comment]
Yeah I should have included "FirstPerson" camera too, I think the issue is the same as long as it's used as a moving 3rd person camera.
(I modified the title of the suggestion)

**Comment by kinsi55** [comment]
I dont know, allowing for that seems like a really bad, clunky solution.

Example: Somebody clicks around in the menu, enables that, now they have a screen in front of their face blocking their view and they cant do anything any more

**Comment by kinsi55** [comment]
If this was something I'd allow it would be a config only option

**Comment by kinsi55** [comment]
The camera screen is wherever the camera is, and depending on where your offset is that can be anywhere :P

**Comment by blackorbit1** [comment]
ah I forgot that it was possible to put an offset from the menu. I was about to suggest making the camera preview disapear when it's too close but it looks like it's already the case for any object in the game ^^
<div align="left">
      <a href="https://www.youtube.com/watch?v=3uO-ILWP_Ic">
         <img src="https://img.youtube.com/vi/3uO-ILWP_Ic/0.jpg" style="width:100%;">
      </a>
</div>

But if the attribute is editable only in the config file, I don't think a lot of peoples could get stuck because of that
